torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -2958,17 +2958,6 @@ def _persistent_reduction_configs(
     # defer to more autotuning, initially
     if "y" in size_hints:
         pass
-<<<<<<< HEAD
-
-    if not max_autotune_enabled: # Don't filter if tuning enabled
```

Comment:
@jataylo to double-check this conflict resolution in case not already consulted

Reply:
Spoke to @naromero77amd, he mentioned that these changes went into rocm7.1_internal_testing but the upstream PR is still open. So, we want to keep rocm7.1_internal_testing changes in place. He pointed me to his upstream PR here: https://github.com/pytorch/pytorch/pull/163908

Tried to keep local changes but some of them were not trivial as Nick's PR upstream is with newer upstream.